src/test/scala-2.11/markets/participants/OrderIssuerSpec.scala

```diff
@@ -17,19 +17,19 @@ package markets.participants
 
 import akka.actor.{ActorRef, ActorSystem}
 import akka.agent.Agent
-import akka.testkit.{TestProbe, TestActorRef, TestKit}
+import akka.testkit.{TestActorRef, TestKit, TestProbe}
 
 import markets.orders.limit.LimitOrderLike
 import markets.tickers.Tick
 import markets.tradables.{Tradable, TestTradable}
 import org.scalatest.{Matchers, GivenWhenThen, FeatureSpecLike}
 
-import scala.collection.{immutable, mutable}
+import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2363134679/issues/source?bid=2921361&fileBranchId=3219434#l27)